src/content/docs/dropins/checkout/checkout-installation.mdx

```diff
@@ -1,9 +1,323 @@
+
+  checkoutProvider.render(OutOfStock, {
+    routeCart: () => '/cart',
+    onCartProductsUpdate: (items) => {
+      cartApi.updateProductsFromCart(items).catch(console.error);
+    },
+  })(outOfStock);
+
+  checkoutProvider.render(LoginForm, {
+    onSignInClick: async (initialEmailValue) => {
+      const signInForm = document.createElement('div');
+
+      authProvider.render(AuthCombine, {
+        signInFormConfig: {
+          renderSignUpLink: true,
+          initialEmailValue,
+          onSuccessCallback: () => {
+            overlaySpinner.classList.add('checkout__overlay-spinner--shown');
+          },
+        },
+        signUpFormConfig: {},
+        resetPasswordFormConfig: {},
+      })(signInForm);
+
+      modal = await createModal([signInForm]);
+      modal.showModal();
+    },
+    onSignOutClick: () => {
+      authApi.revokeCustomerToken();
+    },
+  })(login);
+
+  checkoutProvider.render(ShippingForm, { hideOnVirtualCart: true })(
+    shippingForm,
+  );
+  checkoutProvider.render(BillToShippingAddress, { hideOnVirtualCart: true })(
+    billToShippingAddress,
+  );
+  checkoutProvider.render(BillingForm)(billingForm);
+
+  checkoutProvider.render(ShippingMethods, {
+    hideOnVirtualCart: true,
+    // onShippingMethodSelect: (shippingMethod) => {},
+    onCheckoutDataUpdate: () => {
+      cartApi.refreshCart().catch(console.error);
+    },
+  })(shippingMethods);
+
+  checkoutProvider.render(PaymentMethods, {
+    // slots: {
+    //   Handlers: {
+    //     checkmo: (_ctx) => {
+    //       const $content = document.createElement('div');
+    //       $content.innerText = 'checkmo';
+    //       _ctx.replaceHTML($content);
+    //     },
+    //   },
+    // },
+  })(paymentMethods);
+    checkoutProvider.render(PlaceOrder, {
+    onPlaceOrder: async () => {
+      overlaySpinner.classList.add('checkout__overlay-spinner--shown');
+
+      try {
+        await checkoutApi.placeOrder();
+      } catch (error) {
+        console.error(error);
+        throw error;
+      } finally {
+        overlaySpinner.classList.remove('checkout__overlay-spinner--shown');
+      }
+    },
+  })(placeOrder);
+
+  /*
+   * Render initial layout and setup event handlers
+   */
+
+  mediaQuery.addEventListener('change', (e) => handleScreenResize(e, block));
+  handleScreenResize(mediaQuery, block);
+
+  events.on('authenticated', handleAuthenticated, { eager: true });
+  events.on('checkout/data', (e) => handleCheckoutData(e, block), {
+    eager: true,
+  });
+  events.on('checkout/customer', handleCheckoutCustomer, { eager: true });
+  events.on('checkout/order', (e) => handleCheckoutOrder(e, block));
+}
+```
+
+Test the Checkout dropin by viewing your checkout page in a browser, or running your local dev or build server. If you see the Checkout dropin rendered in the browser, congrats!, you have a successful installation. If not, check the console for any errors and verify that you have followed all the steps correctly.
```

Comment:
If they DID follow instructions, but still have errors, what do they do? Should we send them to FAQ?

Reply:
We don't currently have an FAQ. Perhaps after we publish #73 (which includes an FAQ), we can point them there. Although I don't recall if we have any info there that would be relevant specifically to installing the checkout dropin.